### PR TITLE
Bugfix/discover pixel perfect main two #2076

### PR DESCRIPTION
**Pull request by oskarth**
Progress towards pixel perfection main discover screen

- Fix icon size and remove badge numbers
- Preview: tighten up margins
- Hide horizontal scrollbar
- Padding for public chats

https://github.com/status-im/status-react/issues/2076

image
image

status: ready

### CODE REVIEW

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/discover/all_dapps/views.cljs [hunk 1]
```diff
@@ -20,15 +20,12 @@
       [react/view styles/dapps-list-item-second-row
        [react/view styles/dapps-list-item-name-container
         [react/view styles/dapps-list-item-avatar-container
-         [react/view (chat-icon/contact-icon-contacts-tab dapp)]]
+         [react/view [chat-icon/contact-icon-view dapp {:size 80}]]]
```
Comment (jeluard) [inline src/status_im/ui/screens/discover/all_dapps/views.cljs:23]: Is that `80` expected to be hardcoded?

File: src/status_im/ui/screens/discover/styles.cljs [hunk 2]
```diff
@@ -165,9 +165,11 @@
    :background-color styles/color-white})
 
 (def all-dapps-flat-list-item
-  {:margin 10
-   :width  100
-   :height 100})
+  {:margin          10
+   :width           100
```
Comment (jeluard) [inline src/status_im/ui/screens/discover/styles.cljs:169]: Will those adapt smaller/ bigger screens?

File: src/status_im/ui/screens/discover/all_dapps/views.cljs (hunk 1 above)
Comment (oskarth) [reply src/status_im/ui/screens/discover/all_dapps/views.cljs:23]: According to design it should be 2x, and 40 is hardcoded for rest of app

File: src/status_im/ui/screens/discover/styles.cljs (hunk 2 above)
Comment (oskarth) [reply src/status_im/ui/screens/discover/styles.cljs:169]: Good question, I think it should be fine but might have to set 2 or 4 rows for other resolutions. Will check in further PRs

**Review by flexsurfer — APPROVED** [review]

**Review by jeluard — APPROVED** [review]

File: src/status_im/ui/screens/discover/all_dapps/views.cljs (hunk 1 above)
Comment (oskarth) [reply src/status_im/ui/screens/discover/all_dapps/views.cljs:23]: Looks OK on iPhone 7Plus and not horrible on iPhone 5s. Might have to go to 2 rows or rethink icon size for small resolutions, will check with QA/design next call.

**Pull request merged**